pkg/cli/api.go

```diff
@@ -14,80 +14,58 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cli // nolint:dupl
+package cli //nolint:dupl
 
 import (
 	"fmt"
 
 	"github.com/spf13/cobra"
 
-	"sigs.k8s.io/kubebuilder/v3/pkg/cli/internal/config"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 )
 
+const apiErrorMsg = "failed to create API"
+
 func (c cli) newCreateAPICmd() *cobra.Command {
-	ctx := c.newAPIContext()
 	cmd := &cobra.Command{
-		Use:     "api",
-		Short:   "Scaffold a Kubernetes API",
-		Long:    ctx.Description,
-		Example: ctx.Examples,
+		Use:   "api",
+		Short: "Scaffold a Kubernetes API",
+		Long: `Scaffold a Kubernetes API.
```

Comment:
We can overwrite the description currently. Will still it possible?

Reply:
Yes, the `UpdateContext` function was renamed to `UpdateMetadata` (because context was conflicting with Ginkgo) and the signature modified a bit. Instead of having a single struct with input and output fields, now it has one with input fields (currently only the command name) and another one with output fields (currently examples and description).